src/java/org/apache/cassandra/repair/messages/RepairMessage.java

```diff
@@ -42,7 +51,12 @@
  */
 public abstract class RepairMessage
 {
+    private enum ErrorHandling { NONE, TIMEOUT, RETRY }
     private static final CassandraVersion SUPPORTS_TIMEOUTS = new CassandraVersion("4.0.7-SNAPSHOT");
+    private static final CassandraVersion SUPPORTS_RETRY = new CassandraVersion("5.1.0-SNAPSHOT");
```

Comment:
nit: Note to set to 5.0.0 if we port this to 5.0?

Reply:
yep, if we do that would change to 5.0.0-snpashot